### PR TITLE
Week 3

### PR DESCRIPTION
**Pull request by emilyb7**
This PR should contain everything for week 3, including
* learning outcomes
* schedule
* projects
* morning challenge
* links to workshops
* recommended resources and APIs

FYI reviewers: We accidentally (!) pushed some of this to master on Sunday. @bradreeder has been informed has let us off the hook ;)

related to #29

### CODE REVIEW

**Comment by emilyb7** [comment]
@skibinska @lucymonie @denesnori approval?

File: coursebook/week-3/README.md [hunk 1]
```diff
@@ -1,4 +1,51 @@
...
+14.00 - 18.00 - projects (stop work at 18:00!)  
+
+#### DAY 5  
+
+10.00 - 11.00 - project code review  
+11.00 - 13.00 - respond to issues    
+-- LUNCHTIME 😋 --  
+14.00 - 15.00 - group stop-go-continue  
+15.00 - 15.20 - Prepare Presentations  
+15.20 - 16.50 - presentations (20 mins per group)
```
Comment (skibinska) [inline coursebook/week-3/README.md]: maybe we should follow the structure of week 1
```
14.00 - 15.00 — Group stop-go-continue (Course Coordinator, current week mentors, week 2 mentors)
15.00 - 16.20 — Presentations (1+ current week mentors, 1+ week 2 mentors)
16.20 - 17.00 — Team stop-go-continue (Course Coordinator)
```

Comment (emilyb7) [inline coursebook/week-3/README.md]: @skibinska sure, that's fine. Can you make the changes?

Comment (skibinska) [inline coursebook/week-3/README.md]: :+1:

File: coursebook/week-3/README.md [hunk 2]
```diff
@@ -1,4 +1,72 @@
...
+
+#### DAY 5  
+
+- 10.00 - 11.00 <br /> Project code review  
+
+- 11.00 - 13.00 <br /> Respond to issues   
+
+-- LUNCHTIME 😋 --
+
+- 14.00 - 15.00 <br /> Group stop-go-continue (Course Coordinator, current week mentors, week 4 mentors)
```
Comment (bradreeder) [inline coursebook/week-3/README.md]: The afternoon might need a little re-formatting. One thing I was considering suggesting, as we don't have an external speaker slot, is that me and @nogainbar @sofiapoh @MarckK do a project demo of Breathe With Me on the Friday to round-off the front-end portion of the course?

File: coursebook/week-3/research-afternoon.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 ## Research afternoon
 
-Research afternoon task to be detailed here.
+We won't be doing research afternoon this week! Instead we have 2 full days of workshops to get everyone ready the for the projects.
```
Comment (bradreeder) [inline coursebook/week-3/research-afternoon.md]: Typo. "get everyone ready *for the* "

File: coursebook/week-3/resources.md [hunk 4]
```diff
@@ -1,19 +1,30 @@
...
+**Open postman:** type 'chrome://apps/' into your chrome address bar
+
+**Use postman:** read about postman in the documentation [here](https://www.getpostman.com/docs/requests).
+
+### Optional
 ##### REST
 - REST and HTTP - http://restcookbook.com/Miscellaneous/rest-and-http/
 - Best practices to build RESTful web services - https://www.infosys.com/digital/insights/Documents/restful-web-services.pdf
 
-##### CORS DEEP DIVE
```
Comment (bradreeder) [inline coursebook/week-3/resources.md]: "DEEP DIVE" lmao 😸

File: coursebook/week-3/README.md (hunk 2 above)
Comment (bradreeder) [inline coursebook/week-3/README.md]: Will need to add speaker slot to Wednesday as well if they come back affirmative.

File: coursebook/week-3/resources.md (hunk 4 above)
Comment (lucymonie) [inline coursebook/week-3/resources.md]: I thought it sounded too scary! It's actually really interesting.

Comment (bradreeder) [inline coursebook/week-3/resources.md]: I don't know why I find that funny -- sorry for my immature use of reviews 😅

File: coursebook/week-3/README.md (hunk 2 above)
Comment (emilyb7) [inline coursebook/week-3/README.md]: @bradreeder I've updated Friday. Any news on the Wednesday?

**Comment by emilyb7** [comment]
I've fixed the type and the schedule, can we merge now?

**Comment by JMurphyWeb** [comment]
@bradreeder looks like this needs a mergin'

**Comment by bradreeder** [comment]
I think we may need to review in light of student feedback potentially on the parallel function morning challenge. I think FAC10 found the waterfall challenge quite intense and likely need clarification on callbacks / asynchronous programming.

Happy to merge in mean time!

File: coursebook/week-3/README.md (hunk 2 above)
Comment (sofiapoh) [inline coursebook/week-3/README.md]: Is this on Weds next week? I'm happy to give a talk at some point about BWM but can't do during the day :(

Comment (bradreeder) [inline coursebook/week-3/README.md]: I was suggesting Friday at 5pm @sofiapoh and was going to write to you and the others soon. Won't go ahead without you obvs!

Comment (sofiapoh) [inline coursebook/week-3/README.md]: Hehe no worries, I'll see if I can maybe leave a bit earlier on next friday. Would definitely love to be around for this :)

**Pull request merged**